packages/react-reconciler/src/__tests__/ReactHooks-test.internal.js

```diff
@@ -1618,8 +1618,7 @@ describe('ReactHooks', () => {
             '   Previous render            Next render\n' +
             '   ------------------------------------------------------\n' +
             `1. ${formatHookNamesToMatchErrorMessage(hookNameA, hookNameB)}\n` +
-            '   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\n' +
-            '    in App (at **)',
```

Comment:
Is the stack frame still available in browsers?

Reply:
Since these matches a subset, by removing the stack frame it matches any stack frame.

These end up having the mentioned internals in them so I didn't want to assert on those.